### PR TITLE
build(aio): upgrade angular and zone.js

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -8,20 +8,23 @@
 # alexeagle - Alex Eagle
 # alxhub - Alex Rickabaugh
 # chuckjaz - Chuck Jazdzewski
+# Foxandxss - Jesús Rodríguez
 # gkalpak - George Kalpakas
 # IgorMinar - Igor Minar
 # jasonaden - Jason Aden
 # kara - Kara Erickson
+# juleskremer - Jules Kremer
```
Comment (gkalpak) [inline .pullapprove.yml]: Wrong aphabetic order (should come before `kara`) 😱

File: .pullapprove.yml [hunk 2]
```diff
@@ -249,10 +252,46 @@ groups:
...
+      files:
+        include:
+          - "aio/content/examples/*"
+          - "aio/content/guide/*"
+          - "aio/content/images/*"
+          - "aio/content/tutorial/*"
+          - "aio/content/file-not-found.md"
+    users:
+      - juleskremer #primary
+      - foxandxss
```
Comment (gkalpak) [inline .pullapprove.yml]: It probably doesn't make a difference for PullApprove, but `Foxandxss` is capitalized :grin:

File: packages/router/src/index.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@
  */
 
 
-export {Data, LoadChildren, LoadChildrenCallback, ResolveData, Route, Routes, RunGuardsAndResolvers} from './config';
+export {Data, LoadChildren, LoadChildrenCallback, ResolveData, Route, Routes, RunGuardsAndResolvers, UrlMatcher, UrlMatchResult} from './config';
```
Comment (gkalpak) [inline packages/router/src/index.ts]: Wrong order (`UrlMatchResult` should come before `UrlMatcher`). CircleCI complains :smiley:

**Review by gkalpak — COMMENTED** [review]
The Travis failure (about node-sass) typically happens when upgrading yarn. I usually solve it by removing `aio/node_modules` from the list of cached folders (in `.travis.yml`) and/or running `npm rebuild node-sass` before `yarn install` (iirc).

File: .pullapprove.yml (hunk 1 above)
Comment (IgorMinar) [inline .pullapprove.yml]: OMG! 😮

File: packages/router/src/index.ts (hunk 3 above)
Comment (IgorMinar) [inline packages/router/src/index.ts]: darn you CI! :)

File: .pullapprove.yml (hunk 2 above)
Comment (IgorMinar) [inline .pullapprove.yml]: done

File: .pullapprove.yml (hunk 1 above)
Comment (IgorMinar) [inline .pullapprove.yml]: done

**Comment by IgorMinar** [comment]
@gkalpak PTAL

**Comment by IgorMinar** [comment]
@gkalpak I blew away the travis cache to see if that was the problem

**Review by gkalpak — APPROVED** [review]
LGTM (as soon as Travis is happy :smiley:)

**Review by jasonaden — APPROVED** [review]
lgtm

**Comment by IgorMinar** [comment]
I'm not going to get lgtm from @mhevery and @alexeagle for this one because they are OOO but I don't think it's controversial enough to block it.

**Review by mhevery — APPROVED** [review]

**Comment by jasonaden** [comment]
Merged manually.

**Pull request closed without merging**